bootstrap.go

```diff
@@ -157,6 +157,7 @@ const (
 		hist_id bigint(64) NOT NULL,
 		distinct_count bigint(64) NOT NULL,
 		null_count bigint(64) NOT NULL DEFAULT 0,
+		avg_col_size double NOT NULL DEFAULT 0,
```

Comment:
I think a better way is to store the total size, not the avg size, because when update the value using the DML, total size is easier to update than avg size.